### DISCUSSION TITLE
Using column headers as search keys

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1138] [Q&A]
I don't know if this is a small bug, but when searching using headers I found this situation:

I went to search all the messages of a person with HTML type
In the Header we can see the words: Tipo and Message-From, so search like:
Message-From:"John Smith" && Tipo:html
As a result, there were zero occurrences.
But when I searched with:
Message-From:"John Smith" && tipo:html
(note that the tipo is now lowercase)
The results came out correct.
So I tested it with:
message-from:"John Smith" && tipo:html
And again came zero occurrences.

So I did some research like:
deleted: true
duplicate:true
name:"xxxx.xx"
category: emails
They also only work if you put lowercase letters

So why Message-From don´t work in lowercase or why the others don´t work like syntax in header?

tks,

Paulo

**Comment by lfcnassif** [comment]
Hi @paulobreim. Search properties/keys are case sensitive. Some of them start with an uppercase letter, others start lowercase, you can check how they start looking at the column manager or at metadata filter panel. The main table column headers always render the first letter uppercase, possibly changing the case. I agree this is misleading, it was done to enhance table columns appearance... Other option is to type the TAB key multiple times when you are writing some search key in the search bar, the application will autocomplete the possible property names properly.

**Reply by paulobreim** [reply]
I confess that I had never paid due attention to this panel, and now that you mention it I went to look and it is excellent! Really good.

thank you

**Reply by lfcnassif** [reply]
Well, maybe the metadata filter panel is one of the key features of the tool, comparing to other tools...
